cvdev/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM hudl/opencv
 
 MAINTAINER Ryan Hotovy <ryan.hotovy@hudl.com>
 
-RUN apt-get -y install gdb
+RUN apt-get install -y gdb
+RUN apt-get update && apt-get install -y libboost-all-dev --fix-missing
```

Comment:
It just occurred to me that we still haven't merged this - Do we want to add boost to the base `hudl/opencv` image instead?